common/src/main/scala/org/apache/spark/sql/comet/util/Utils.scala

```diff
@@ -69,6 +69,7 @@ object Utils {
     case int: ArrowType.Int if int.getIsSigned && int.getBitWidth == 8 * 2 => ShortType
     case int: ArrowType.Int if int.getIsSigned && int.getBitWidth == 8 * 4 => IntegerType
     case int: ArrowType.Int if int.getIsSigned && int.getBitWidth == 8 * 8 => LongType
+    case int: ArrowType.Int if int.getBitWidth == 8 * 8 => LongType
```

Comment:
Hmm, is this UInt64? Using `LongType` to represent it will overflow, I think.

Reply:
Shall we map to `DecimalType` instead?

Reply:
Where do you use it?

Reply:
This `UInt64` is for state field `count`.

Reply:
If we have both partial and final aggregation operators in Comet, it should be okay as Java doesn't process the intermediate results (state), but if we have only partial aggregation in Comet, this Uint64 array as LongType will possibly cause overflow in corner case, I think.

Reply:
Shall we map to `DecimalType(20, 0)` instead?

Reply:
Hmm, how does it work? You will treat UInt64 array as decimal array?

Reply:
Since the max of UInt64 is 18446744073709551615, I guess we can use `DecimalType(20, 0)` to represent the number without overflowing?

Reply:
Yes, but I mean Spark will treat the actual UInt64 array as decimal one. For example, it will call `getDecimal` on an UInt64 array, does it work?

Reply:
Maybe we should enable partial + final Comet aggregation as a whole, See #223.